src/api/player.js

```diff
@@ -47,6 +47,11 @@ function Player(selector, contentInfo) {
   // Expose a public .isPaused attribute.
   this.isPaused = false;
 
+  // Other public attributes
+  this.time_ = {currentTime: 0, duration: 0};
```

Comment:
Stick to the convention established in the rest of the file (this.time instead of this.time_)